src/api/client.ts

```diff
@@ -142,6 +142,13 @@ async function createContractGroup(
     return { error }
 }
 
+async function getContractGroup(group: string) {
```

Comment:
Any reason we aren't 
1) Returning just the `data` object as a whole unless there's an error like in some of the other client functions, and then...
2) Defining that return value as a type like in the other client functions
?